protoc/StrUtil.cpp

```diff
@@ -360,39 +360,39 @@ char* FastUInt32ToBufferLeft(uint32_t u, char* buffer) {
     buffer[0] = ASCII_digits[0];
     buffer[1] = ASCII_digits[1];
     buffer += 2;
- sublt100_000_000:
+  sublt100_000_000:
```

Comment:
Ironic that this is an (old) Google file, or maybe it's cpplint not handling partially indented labels?

Reply:
That's interesting. Should I undo these indent fixes and add NOLINTs or leave them fixed like this?

Reply:
No I think leaving them fixed is fine, I'm sort of curious if the CPP styleguide has moved on, or it just isn't handling some edge cases around labels.